ax/plot/table_view.py

```diff
@@ -158,4 +159,5 @@ def transpose(m):
         margin=go.layout.Margin(l=0, r=20, b=20, t=20, pad=4),  # noqa E741
     )
     fig = go.Figure(data=[trace], layout=layout)
+    # pyre-fixme[7]: Expected `Tuple[DataFrame]` but got `AxPlotConfig`.
```

Comment:
Is the inferred type wrong then?

Reply:
This sometimes returns a tuple of dfs and sometimes returns an AxPlotConfig... Just not a great function I suppose